middleware/http/opa/opa_middleware_test.go

```diff
@@ -0,0 +1,113 @@
+package opa
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/dapr/components-contrib/middleware"
+	"github.com/dapr/dapr/pkg/logger"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	fh "github.com/valyala/fasthttp"
+)
+
+func mockedRequestHandler(ctx *fh.RequestCtx) {}
+
+func TestBasicOpaPolicy(t *testing.T) {
```

Comment:
These tests are very similar in structure. May be consider writing using table-driven test https://dave.cheney.net/2019/05/07/prefer-table-driven-tests and reuse much of the skeleton? Thanks!

Reply:
Oooo, that's neat. I'll get the tests converted, certaintly a lot of duplication hapening.